shell/common/rasterizer.cc

```diff
@@ -156,6 +156,12 @@ void Rasterizer::NotifyLowMemoryWarning() const {
   context->performDeferredCleanup(std::chrono::milliseconds(0));
 }
 
+void Rasterizer::CollectView(int64_t view_id) {
+  if (view_id == kFlutterImplicitViewId) {
```

Comment:
Since this logic will need to be updated for multi-view, please add a TODO comment and assert that `view_id` is `kFlutterImplicitViewId`